engineio/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "rust_engineio"
+version = "0.2.4"
+authors = ["Bastian Kersting <bastian@cmbt.de>"]
+edition = "2018"
+description = "An implementation of a engineio client written in rust."
+readme = "README.md"
+repository = "https://github.com/1c3t3a/rust-socketio"
+keywords = ["engineio", "network", "protocol", "client"]
+categories = ["network-programming", "web-programming", "web-programming::websocket"]
+license = "MIT"
+
+[dependencies]
+base64 = "0.13.0"
+bytes = "1"
+rand = "0.8.1"
```

Comment:
Do we need rand here? The project actually compiles without it. Are there any other dependencies we could sort out?

Reply:
Looks like rand is only used in socketio [here](https://github.com/1c3t3a/rust-socketio/blob/858d236b0337c15f8a9601355336e9b05f99167d/socketio/src/client/socket.rs#L4)

> Are there any other dependencies we could sort out?

https://github.com/1c3t3a/rust-socketio/pull/116